app/scenes/GroupNew.js

```diff
@@ -1,90 +1,90 @@
 // @flow
-import { observable } from "mobx";
-import { inject, observer } from "mobx-react";
+import { observer } from "mobx-react";
 import * as React from "react";
-import { withRouter, type RouterHistory } from "react-router-dom";
+import { useTranslation, Trans } from "react-i18next";
 import GroupsStore from "stores/GroupsStore";
-import UiStore from "stores/UiStore";
 import Group from "models/Group";
 import GroupMembers from "scenes/GroupMembers";
 import Button from "components/Button";
 import Flex from "components/Flex";
 import HelpText from "components/HelpText";
 import Input from "components/Input";
 import Modal from "components/Modal";
+import useStores from "hooks/useStores";
 
 type Props = {
-  history: RouterHistory,
-  ui: UiStore,
   groups: GroupsStore,
   onSubmit: () => void,
 };
 
-@observer
-class GroupNew extends React.Component<Props> {
-  @observable name: string = "";
-  @observable isSaving: boolean;
-  @observable group: Group;
+function GroupNew({ groups, onSubmit }: Props) {
+  const { ui } = useStores();
```

Comment:
```suggestion
  const { ui, groups } = useStores();
```

Reply:
The functionality of this component is currently broken as groups is undefined

Reply:
Thanks! I feel so STUPID.

Reply:
This is exactly the kind of thing that better typing should catch